### PR TITLE
netcoreapp2.0 packages are compatibile with netcoreapp2.1

### PR DESCRIPTION
**Pull request by enricosada**
the `netcoreapp2.1` framework support `netcoreapp2.0` packages

### CODE REVIEW

File: src/Paket.Core/Versioning/FrameworkHandling.fs [hunk 1]
```diff
@@ -612,7 +612,7 @@ type FrameworkIdentifier =
         | DotNetCoreApp DotNetCoreAppVersion.V1_0 -> [ DotNetStandard DotNetStandardVersion.V1_6 ]
         | DotNetCoreApp DotNetCoreAppVersion.V1_1 -> [ DotNetCoreApp DotNetCoreAppVersion.V1_0 ]
         | DotNetCoreApp DotNetCoreAppVersion.V2_0 -> [ DotNetCoreApp DotNetCoreAppVersion.V1_1;  DotNetStandard DotNetStandardVersion.V2_0 ]
-        | DotNetCoreApp DotNetCoreAppVersion.V2_1 -> [ DotNetCoreApp DotNetCoreAppVersion.V1_1;  DotNetStandard DotNetStandardVersion.V2_0 ]
+        | DotNetCoreApp DotNetCoreAppVersion.V2_1 -> [ DotNetCoreApp DotNetCoreAppVersion.V2_0;  DotNetStandard DotNetStandardVersion.V2_0 ]
```
Comment (matthid) [inline src/Paket.Core/Versioning/FrameworkHandling.fs]: I guess we can remove dotnetstandard version here we 2.0 already supports that

Comment (enricosada) [reply src/Paket.Core/Versioning/FrameworkHandling.fs]: sure, let me add a test `netcoreapp2.1`+`netstandard2.0` and i'll remove it

**Comment by matthid** [comment]
I'm curious: Did you notice an impact of this? It looks like something incredible hard to debug, but I can't think of what issues would arise from this...

**Comment by enricosada** [comment]
@matthid added cleaned up + test

i found it working on my next PR, i just splitted it so is easier to review.
The issue was a target framework condition was not populated correctly, but yes, is really small impact (really few `netcoreapp2.0` packages out there)

**Comment by matthid** [comment]
Looks good to me. Thanks for the tests!

**Comment by matthid** [comment]
> I'll merge so when build is green.

Good luck with that ;)

**Comment by forki** [comment]
it's rather important fix. I'll release. Thanks!

**Comment by matthid** [comment]
Honestly I'm a bit worried that our reviews didn't catch this, but good that it is solved now.

**Pull request merged**